rs/execution_environment/src/execution_environment.rs

```diff
@@ -633,7 +636,7 @@ impl ExecutionEnvironment {
                         Ok(args) => {
                             let key_id = MasterPublicKeyId::Ecdsa(args.key_id.clone());
                             match get_master_public_key(
-                                idkg_subnet_public_keys,
+                                chain_key_subnet_public_keys,
```

Comment:
I think the design document says we should rename `idkg_subnet_public_keys` to `threshold_subnet_public_keys`, but I don't have a strong opinion here cc @fspreiss

Reply:
This was captured in a different ticket (CRP-2616). I decided to do the renaming in a separate PR